### ISSUE TITLE
Can you provide some details about your training dataset?

### ISSUE THREAD

**Issue by windpls** [qiexing/face-landmark-localization#1]
It's very kind of you to share the model.

But I wonder some details about the training set. Did you use external dataset apart from IBUG 300W? If used, what's the size of your private dataset?   If not, how did you avoid overfitting while using such small data set?

**Comment by qiexing** [comment]
hi, windpls. I just using the data from ibug 300w without external dataset. I do rotations, translation to avoid overfitting.

**Comment by windpls** [comment]
@qiexing Thanks for your reply. Have you tested the model on the test set of IBUG 300W (IBUG + test set of HELEN + test set of LFPW, totally 689 samples)? Does the performance outperform previous regression-based methods like ERT, LBF, SDM?

**Comment by qiexing** [comment]
If using ground truth point to calculate bbox, the result would be better

**Comment by windpls** [comment]
@qiexing Thanks. Another thing I'd like to know is whether the model you released in this repo was trained on the whole IBUG300W or just the training set(not including the 689 samples I mentioned above)?

**Comment by qiexing** [comment]
not including 689 test samples
